cfgov/jinja2/v1/browse-filterable/index.html

```diff
@@ -21,16 +21,17 @@
         </div>
     {% endfor %}
     {% for block in page.content %}
-        <div class="block
+        {% if 'filter_controls' in block.block_type %}
```

Comment:
Could this condition be removed if you gave `filter_controls` the same modifier?